docs/ABOUT.md

```diff
@@ -0,0 +1,14 @@
+JavaScript is a great language, but not everyone likes writing in it. Thus, people and organizations have started to write languages that transpile(get changed from one syntax to another) into JavaScript, that they feel are nicer to write in than JavaScript. CoffeeScript is one example, another is Microsoft's TypeScript.
+
+What are the specific advantages of CoffeeScript? From [coffeescript.org](http://www.coffeescript.org):
+
+> CoffeeScript is a little language that compiles into JavaScript. Underneath that awkward Java-esque patina,
+> JavaScript has always had a gorgeous heart. CoffeeScript is an attempt to expose the good parts of JavaScript
+> in a simple way.
+>
+> The golden rule of CoffeeScript is: "It's just JavaScript". The code compiles one-to-one into the equivalent JS,
+> and there is no interpretation at runtime. You can use any existing JavaScript library seamlessly from CoffeeScript
+> (and vice-versa). The compiled output is readable and pretty-printed, will work in every JavaScript runtime, and tends
+> to run as fast or faster than the equivalent handwritten JavaScript.
+
+You should learn CoffeeScript if you like what you get out of JavaScript, but don't like what you have to put in.
```

Comment:
The second half of this sentence suggests that if you _do_ like JavaScript, you shouldn't learn CoffeeScript. I think that people could get a lot out of learning things that transpile to JavaScript, even if they like JavaScript.